README.md

```diff
@@ -99,7 +99,7 @@ one page for the item details; in the example, the detail page for Czech Republi
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://sp-metrics-webapp.netlify.app)
```

Comment:
Your deployment link is still broken. Kindly check your Netlify deploy settings to make sure that your application is deployed correctly.

![Screenshot from 2023-01-21 09-48-17](https://user-images.githubusercontent.com/38088818/213850045-c5975df4-f234-4a0a-91b1-c2c4fc270d68.png)

Reply:
@KDlamini, i have kindly updates the rename, and added more unit/integration tests.  the site is deployed on main branch